QUICKSTART-ECS.md

```diff
@@ -0,0 +1,191 @@
+# Using a Bottlerocket AMI with Amazon ECS
+
+> The [ECS variant](./aws-ecs.md) of Bottlerocket is currently in a developer preview phase and we're looking for your
+> [feedback](https://github.com/bottlerocket-os/bottlerocket#contact-us).
+> We'd love for you to try it out!
+
+[Amazon Elastic Container Service (Amazon ECS)](https://ecs.aws) is a highly scalable, fast, container management service that makes it easy to run, stop, and manage containers on a cluster.
+Your containers are defined in a task definition which you use to run individual tasks or as a service.
+
+This quickstart will walk through setting up an Amazon ECS cluster with Bottlerocket container instances (using the EC2 launch type).
+Check out the [Amazon ECS developer guide](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html) for more information on the supported launch types, additional details, and a broad overview of Amazon ECS.
+
+## Prerequisites
+
+Before you begin, be sure that you've completed the steps in
+[Setting up with Amazon ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/get-set-up-for-amazon-ecs.html)
+and that your AWS user has either the permissions specified in the `AdministratorAccess` or the
+[Amazon ECS First Run Wizard Permissions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/security_iam_id-based-policy-examples.html#first-run-permissions)
+IAM policy example.
+
+## Create a cluster
+
+An Amazon ECS cluster is a logical grouping of tasks, services, and container instances.
+For more information about clusters, see
+[Amazon ECS clusters](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/clusters.html).
+
+You can create a cluster with the AWS CLI as follows:
+
+```
+aws ecs --region us-west-2 create-cluster --cluster-name bottlerocket
+```
+
+> Note: The command above and subsequent examples include the AWS region, so change it from `us-west-2` if you operate in another region.
+
+## Finding an AMI
```

Comment:
I think when official ECS AMIs are published, we should move this into a separate doc, shared with the k8s quickstart.